numba/npyufunc/parfor.py

```diff
@@ -263,13 +422,29 @@ def _hoist_internal(inst, dep_on_param, call_table, hoisted, typemap):
             print("Instruction", inst, " could not be hoisted because it isn't pure.")
     return False
 
+def find_setitems_block(setitems, block):
```

Comment:
Merge master since hoisting PR is merged.

Reply:
Done.